common/src/main/java/io/netty/util/ConstantPool.java

```diff
@@ -31,6 +31,7 @@
 
     private final ConcurrentMap<String, T> constants = PlatformDependent.newConcurrentHashMap();
 
+    @Deprecated
```

Comment:
@doom369 yeah... the only thing I said was that you not need to mark the private field as `@Deprecated`. Just remove it here :)